app/src/main/java/org/mozilla/fenix/wallpapers/WallpapersUseCases.kt

```diff
@@ -258,6 +258,9 @@ class WallpapersUseCases(
             } else {
                 settings.currentWallpaperName
             }
+            if (settings.shouldMigrateLegacyWallpaperCardColors) {
+                migrationHelper.migrateExpiredWallpaperCardColors()
+            }
```

Comment:
nit: could we add a newline after this brace and possibly before `val currentWallpaperName = ...` to visually separate the migration code?